buildkite/bazelci.py

```diff
@@ -81,6 +81,11 @@
     "bazel": "gs://bazel-kzips/",
 }[BUILDKITE_ORG]
 
+# Set this to 'true' to run bazelci in local-only mode, with no attempt to
+# use remote cache/execution, and no attempt to upload test results.
+# Only compatible with `bazelci.py runner`, not other subcommands.
+BAZELCI_LOCAL_RUN = os.environ.get("BAZELCI_LOCAL_RUN", "false") == "true"
```

Comment:
Can you switch to use `is_trueish` ?
https://github.com/bazelbuild/continuous-integration/blob/308097df57a6e1dd4cab68bfff5ba48764ddb82f/buildkite/bazelci.py#L1163C5-L1164

Reply:
Done. I also changed to a helper method to match `use_bazelisk_migrate`.

Reply:
Perfect, thank you!